tests/test_utils_package.py

```diff
@@ -1,6 +1,6 @@
 from textwrap import dedent
 from rsconnect.utils_package import (
-    _remove_leading_zeros, # pyright: ignore[reportPrivateUsage]
+    _remove_leading_zeros,  # pyright: ignore[reportPrivateUsage]
```

Comment:
aside: This adjustment was needed with the previous version of black, as well. We didn't see it because lint problems go unseen in CI (likely because fmt is run before lint).